pkgs/development/python-modules/manticore/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, buildPythonPackage
+, capstone
+, crytic-compile
+, fetchFromGitHub
+, pkgs
+, ply
+, prettytable
+, pyelftools
+, pyevmasm
+, pysha3
+, pytestCheckHook
```

Comment:
I personally prefer to have ``pytestCheckHook`` towards the bottom, in a separate "Test Inputs" section.